pandora-client-web/src/ui/screens/spaceConfiguration/spaceConfiguration.tsx

```diff
@@ -191,7 +191,7 @@ export function SpaceConfiguration({ creation = false }: { creation?: boolean; }
 				{ canEdit && !IsValidName(currentConfig.name) ? (<div className='error'>Invalid name</div>) : null }
 			</div>
 			<div className='input-container'>
-				<label>Space size (maximum number of characters allowed inside)</label>
+				<label>Space size (maximum number of characters allowed inside - from 1 to 100)</label>
```

Comment:
You should use `LIMIT_SPACE_MAX_CHARACTER_NUMBER` instead of hard-coding `100`.

Reply:
Fixed~